vpr/src/base/vpr_types.h

```diff
@@ -1143,6 +1143,26 @@ struct t_placer_opts {
     bool enable_analytic_placer;
 };
 
+
+/******************************************************************
+ * Analytical Placer data types
+ *******************************************************************/
+
+/**
+ * @brief Various options for the Analytical Placer.
+ *
+ *   @param doAnalyticalPlacement
+ *              True if analytical placement is supposed to be done in the CAD
+ *              flow. False if otherwise.
+ */
+struct t_ap_opts {
```

Comment:
I know we've got lots of types in this file already, but I wonder if the t_ap_ops should go in the analytical_place directory (maybe ap_types.h) instead?  Don't worry about it if that would be a big refactor or be more trouble than it is worth due to being inconsistent with other opts.

Reply:
This struct is actually part of the VPR_Setup struct and is used to pass command-line arguments to the flows. I think it would make more sense if all of these structs were moved to a better place; separating them out to their respective flows seems confusing to me. I think its better to wait on this and do this all at once.